internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -456,7 +464,7 @@ func (r *Reconciler) reconcileExporterQueriesConfig(ctx context.Context,
 		return nil, err
 	}
 
-	if !pgmonitor.ExporterEnabled(cluster) {
+	if !pgmonitor.ExporterEnabled(cluster) || feature.Enabled(ctx, feature.OpenTelemetryMetrics) {
```

Comment:
Just a note: as of now, we only run `addPGExporterToInstancePodSpec` in instance.go (line 1227) when the metrics feature gate is not enabled, right?

Reply:
That's right.